index.html

```diff
@@ -5,5 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <input type="text" />
```

Comment:
```suggestion
    <input type="text" class="search-bar"  />
```
- input에 클래스가 추가되어야 할 것 같아요 👍